### PR TITLE
Second CRUD

### PR DESCRIPTION
**Pull request by DarshanAIakhramsing**
Plox take a look

### CODE REVIEW

File: database/seeds/RoleHasPermissionsSeeder.php [hunk 1]
```diff
@@ -11,7 +11,7 @@ class RoleHasPermissionsSeeder extends Seeder
      */
     public function run()
     {
-        for($i=1;$i<=17;$i++) {
+        for($i=1;$i<=21;$i++) {
```
Comment (okapifan) [inline database/seeds/RoleHasPermissionsSeeder.php]: maak 22

**Review by okapifan — CHANGES_REQUESTED** [review]
Go fix

**Review by okapifan — APPROVED** [review]
Well done slave

**Review by Kokosnootje — APPROVED** [review]
Ya did gud kid

**Pull request merged**